websites/T/TruckersFM/presence.ts

```diff
@@ -1,16 +1,31 @@
 const presence = new Presence({
   clientId: "640914619082211338"
-});
-
+}), browsingStamp = Math.floor(Date.now() / 1000);
 presence.on("UpdateData", () => {
   const presenceData: PresenceData = {
-    largeImageKey: "tfmlogo"
+    largeImageKey: "tfmlogo",
+	startTimestamp: browsingStamp
   };
 
   presenceData.details = `${document.querySelector(".player-artist-text").textContent} - ${document.querySelector(".player-title-text").textContent}`;
   presenceData.state = document.querySelector(".live-name").textContent
     ? document.querySelector(".live-name").textContent
     : 'AutoDJ';
+	
+  presenceData.buttons = [
+	{
+		label: 'Tune into TFM',
+		url: 'https://truckers.fm'
```

Comment:
![image](https://user-images.githubusercontent.com/9348108/112150753-bde88700-8be0-11eb-8d36-2a80cbfd7b22.png)

Reply:
You can redirect users to: ``https://truckers.fm/listen``

Reply:
Changed to /listen